package.json

```diff
@@ -5,9 +5,9 @@
     "test": "npx lerna exec npm t",
     "build": "npx lerna run build",
     "docs": "npx typedoc",
-    "publish:release-candidate": "npx lerna publish --allow-branch release-candidate --dist-tag next --preid rc --no-verify-access --conventional-prerelease bump preminor",
-    "publish:release": "npx lerna publish --allow-branch main --dist-tag latest --no-verify-access --conventional-graduate bump minor",
-    "publish:hotfix": "npx lerna publish --allow-branch $(git rev-parse --abbrev-ref HEAD) --no-verify-access --conventional-graduate bump patch",
+    "publish:release-candidate": "npx lerna publish --dist-tag next --preid rc --no-verify-access --conventional-prerelease bump preminor --yes",
```

Comment:
can we leave the `--yes` out of the command so it still asks for confirmation when running manually, but have the github action pass in `--yes` as an additional argument?

Reply:
--yes ;)